ui/components/FlipCard/FlipCard.style.js

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+
+export const Container=styled('div')(({theme})=>({
+  position:"relative",
+  height:"100%",
+  borderRadius : theme.spacing(1),
+}))
+
+export const InnerCard=styled('div')(({flipped,theme})=>({
+      padding : theme.spacing(2),
+      borderRadius : theme.spacing(1),
+      position:"relative",
+      height:"100%",
```

Comment:
Also remove the height, as when we flip a card the  height of other cards also increase.